src/test/node/user.js

```diff
@@ -1,9 +0,0 @@
-import test from 'tape';
```

Comment:
insertMockData method occurs an travis failure in un-updated code (like mine)
and will be moved to somewhere.

so I removed it